src/ConsoleApp/consoleAppDemo.java

```diff
@@ -6,16 +6,90 @@
 
 public class consoleAppDemo {
 	
+	static int console[] = {1, 2, 3,};
+	int menuoptions; 
+	static int menuinput;
 
 	public static void main(String[] args) {
-
-		System.out.println("Brewmaster");
 		
+	int console[] = {1, 2, 3,};
+		
+		Scanner in = new Scanner(System.in);
+		System.out.println("Brewmaster");
 		System.out.println("Press enter to continue");
+		
+			in.nextLine();
+		System.out.println("Press (1) for open Tavern Owners Menu");
+		System.out.println("Press (2) for view Inventory");
+		System.out.println("Press (3) for Admin Menu");
+		System.out.println("press (4) to Close Application");
+		System.out.println("  ");
+		System.out.println("ENTER YOUR OPTION    :");
+	{
+	
+try	{	
+	while (menuinput==0)
```

Comment:
You are on the right track. I recommend: Move all of your println statements up  for the menu above into the while loop. Get rid of the 'for' loop and the 'console' array. You don't need to parse the numbers, they can stay as strings. Use a nested-if block to compare the user input against the numbers. Don't use a try catch here - try catch exists to catch thrown exceptions. Nothing here is throwing an exception to catch - You don't need any exception handling for this card.

Reply:
understood and appreciated.
Have instead included a switch statement for ease of expansion later on

have run into difficulty with ending the program.